### ISSUE TITLE
Make Login/Signup/Forgot-Password Pages "Sveltey"

### ISSUE THREAD

**Issue by scosman** [CriticalMoments/CMSaasStarter#6]
The login pages use `@supabase/auth-ui-svelte` but despite the name, these are wrappers of a general javascript framework, that doesn't produce Svelte. They can't be pre-rendered, are a bit slower to load, and show a visual stutter when applying style. I also don't love the error message handling (shown below, not inline), and the "in progress" indicator (too subtle, and can be below the viewport on mobile).

We can re-implement these in pure svelte for pre-render/consistency/speed, and fix the UX issues mentioned above.

P2: everything is functional and works fine as is.

Open for community input on this one.

**Comment by scosman** [comment]
The supabase team is tracking the root issue here: https://github.com/supabase/auth-ui/issues/230

I have a workaround, but will see if they fix it soon as it’s not critical. 

Would still like svelte-rendering instead of that library so leaving open as P2.

**Comment by scosman** [comment]
In integrated my workaround in https://github.com/CriticalMoments/CMSaasStarter/commit/9211be0afccc82aa3916dcd9206a204ee092d742 . Looks like no ETA on real fix, and easy enough to revert if they do.

Closing the issue. While I wouldn't mind moving away `@supabase/auth-ui-svelte` for performance, I realized it would be a ton of work. It's loading in all the oAuth options and icons. It would need to be kept up to date as Supabase adds/removes providers. Best to use the official solution for long term stability.
